teachertool/src/transforms/removeCriteriaFromChecklist.ts

```diff
@@ -14,7 +14,10 @@ export function removeCriteriaFromChecklist(criteriaInstanceId: string) {
 
     const newChecklist = {
         ...teacherTool.checklist,
-        criteria: teacherTool.checklist.criteria.filter(c => c.instanceId !== criteriaInstanceId),
+        criteria: teacherTool.checklist.criteria.filter(c =>
+            c.instanceId !== criteriaInstanceId ||
+            c.instanceId === criteriaInstanceId && !c.deleted
```

Comment:
I think the `c.instanceId === criteriaInstanceId` check is redundant. Could just be `c.instanceId !== criteriaInstanceId || !c.deleted`